scanpy/preprocessing/simple.py

```diff
@@ -591,34 +593,36 @@ def pca(data, n_comps=None, zero_center=True, svd_solver='auto', random_state=0,
         logg.msg('reducing number of computed PCs to',
                n_comps, 'as dim of data is only', adata.n_vars, v=4)
 
+    adata_hvg = adata[:, adata.var['highly_variable']] if use_hvg and 'highly_variable' in adata.var.keys() else adata
```

Comment:
If use_hvg is False, this naming doesn't make sense. I would keep it as adata even if use_hvg is True.

Reply:
It just occurred to me that `keep_all_genes` in a function that is called `filter_genes_...` is not awesome.

Can we do, `subset=True` or `do_subset=True` or `merely_annotate=False` or something like this?